lib/physicians/validation.ts

```diff
@@ -64,6 +64,13 @@ export const editPhysicianAssessmentSchema = object({
   physicianAssessment: physicianAssessmentSchema,
 });
 
+/**
+ * Validation schema for physician assessment mutation
+ */
+export const physicianAssessmentMutationSchema = physicianAssessmentSchema.shape({
+  id: number().positive('Application not found').required('Application not found'),
```

Comment:
Please see my comment here https://github.com/uwblueprint/richmond-centre-for-disability/pull/205#discussion_r884309096

Reply:
Addressed!